### PR TITLE
vendor all deps in the repo, rewrite imports.

### PR DESCRIPTION
**Pull request by aybabtme**
This way, the tool can be trully just `go get`'d.

@phillbaker @bryanl

### CODE REVIEW

File: droplet.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func dropletCreate(ctx *cli.Context) {
 		UserData:          userData,
 	}
 
-	dropletRoot, resp, err := client.Droplets.Create(createRequest)
+	droplet, resp, err := client.Droplets.Create(createRequest)
```
Comment (phillbaker) [inline droplet.go]: Yup, thanks for bumping godo too.

File: Godeps/Godeps.json [hunk 2]
```diff
@@ -7,19 +11,29 @@
...
 		{
-			"ImportPath": "github.com/jmcvetta/napping",
-			"Rev": "5292ff1b0132449a9a928004afd03f23be9d750d"
+			"ImportPath": "github.com/digitalocean/godo",
+			"Rev": "8eb611fd0cd8407c6341dafe28659218a1c42483"
 		},
 		{
-			"ImportPath": "gopkg.in/yaml.v1",
-			"Rev": "feb4ca79644e8e7e39c06095246ee54b1282c118"
+			"ImportPath": "github.com/google/go-querystring/query",
```
Comment (phillbaker) [inline Godeps/Godeps.json]: Are adding these intentional?

Comment (aybabtme) [inline Godeps/Godeps.json]: they're dependencies of dependencies, i guess

**Comment by phillbaker** [comment]
:+1:

**Pull request merged**